lib/classes/PluginManager.js

```diff
@@ -104,9 +104,15 @@ class PluginManager {
 
         this.addPlugin(Plugin);
       } catch (error) {
-        this.serverless.cli.log(`WARNING: Serverless plugin "${plugin}" not found.
+        const errorMessage = `WARNING: Serverless plugin "${plugin}" not found.
 
-Make sure it's installed and listed correctly in plugins section in 'serverless.yml' file.`);
+  Make sure it's installed and listed correctly in plugins section in 'serverless.yml' file.`;
+
+        if (!this.cliOptions.help) {
+          throw new this.serverless.classes.Error(errorMessage);
```

Comment:
In this case I would prepend the message with "ERROR", not "WARNING".
Maybe you can define the base message without the severity text and add it when logging the warning or throwing the error.